libs/shared/models/src/lib/hackathon.model.ts

```diff
@@ -36,6 +36,9 @@ export interface IHackathon {
   community: ICommunity;
   community_group: ICommunityGroup;
   total_prize_amount: object;
+  kommunity_slug?: string;
+  interested_members_count?: number;
+  start_time?: Date;
```

Comment:
let's remove kommunity_slug and make it deprecate soon!, indtead use community.slug
don't use start_time, instead use start_date